inspirehep/utils/record.py

```diff
@@ -143,6 +144,13 @@ def soft_delete_pidstore_for_record(record_id):
     pids = PersistentIdentifier.query.filter(PersistentIdentifier.object_uuid == record_id).all()
 
     for pid in pids:
-        deleted = pid.delete()
+        pid.delete()
+
+
+def merge_pidstores_of_two_merged_records(record_merged_uuid, record_deleted_uuid):
+    """Merge all pidstores of deleted record to the merged record."""
+    pids_deleted = PersistentIdentifier.query.filter(PersistentIdentifier.object_uuid == record_deleted_uuid).all()
+    pids_merged = PersistentIdentifier.query.filter(PersistentIdentifier.object_uuid == record_merged_uuid).one()
 
-    return deleted
```

Comment:
I've done it in a separate commit. It was useless because the returned value is not used somewhere. :smile: